src/api.js

```diff
@@ -624,9 +671,9 @@ const history: HandlerFunction = async function (req, _res) {
       hash: tx.id,
       time: iso8601date,
       tx_state: 'Successful',
-      inputs: tx.inputs,
-      dataInputs: tx.dataInputs,
-      outputs: tx.outputs,
+      inputs: input.concise ? undefined : tx.inputs,
```

Comment:
This might be confusing since I wouldn't expect a variable to be assigned to `undefined` explicitly. I would expect `null` instead since `undefined` means there is no such variable, but `null` means the variable isn't filled. Also assigning `undefined` to something is weird since it is not a real value.

Reply:
You are right, sorry about that, fixed!